pkgs/top-level/python-packages.nix

```diff
@@ -23813,6 +23782,8 @@ EOF
 
   wheel = callPackage ../development/python-modules/wheel { };
 
+  whitey = callPackage ../applications/video/whitey { };
```

Comment:
This is an application and should therefore be called directly from `all-packages.nix`, and not via `python-packages.nix`.